### PR TITLE
Update doctorfatekentnelson.ms

### PR DESCRIPTION
**Pull request by flamingwarrior10**
Power Remake for Dr.Fate
scroll down to powers for Dr.Fate
http://dc.wikia.com/wiki/Kent_Nelson_(New_Earth)

### CODE REVIEW

**Comment by flamingwarrior10** [comment]
used wiki on accident http://comicvine.gamespot.com/doctor-fate-kent-nelson/4005-6599/ more info

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 1]
```diff
@@ -9,6 +9,23 @@
             'trail': '',
             'op': 'true',
             'speech': "I am Kent Nelson, a.k.a. Doctor Fate.",
-            'powers': array(),
-            'buffs': array()
+            'powers': array(
+                "/buff",
+                "/thru (intangible)",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: EVIDENCE REQUIRED: intangibility

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 2]
```diff
@@ -9,6 +9,23 @@
...
+                "/invis",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: EVIDENCE REQUIRED: invisibility

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 3]
```diff
@@ -9,6 +9,23 @@
...
+                "/gmc",
+                "/gms",
+                "/tp",
+                "/fly"
+                "/heal",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: EVIDENCE REQUIRED: healing self

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 4]
```diff
@@ -9,6 +9,23 @@
...
+                "/run 0-4",
+                "/jump 0-3"
+
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: where is haste?

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 5]
```diff
@@ -9,6 +9,23 @@
...
+                "/gms",
+                "/tp",
+                "/fly"
+                "/heal",
+                "/run 0-4",
+                "/jump 0-3"
+                
+            ),    
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: EVIDENCE REQUIRED: punching as hard as superman

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 6]
```diff
@@ -9,6 +9,23 @@
...
+                "/effect ". @plr ." 10 9999 5 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: EVIDENCE REQUIRED: extremely fast regen

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 7]
```diff
@@ -9,6 +9,23 @@
...
+                "/effect ". @plr ." 21 9999 12 true",
+                "/effect ". @plr ." 16 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: EVIDENCE REQUIRED: night vision

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms (hunk 4 above)
Comment (flamingwarrior10) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: no haste

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 8]
```diff
@@ -9,6 +9,20 @@
...
-            'buffs': array()
+            'powers': array(
+                "/buff",
+                "/gmc",
+                "/gms",
+                "/tp",
+                "/fly"
+                "/run 0-4",
+                "/jump 0-3"
+
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: delete empty line

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 9]
```diff
@@ -9,6 +9,18 @@
...
+                "/gms",
+                "/tp",
+                "/fly"
+                "/run 0-4",
+                "/jump 0-3"                
+            ),    
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 5 true",
+                "/effect ". @plr ." 11 9999 1 true",
+                "/effect ". @plr ." 21 9999 4 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: delete comma

**Pull request merged**